open_fdd/validation/bench_5007_long_fdd.py

```
@@ -0,0 +1,1311 @@
+            event_type="equivalence_check",
+            message=(
+                f"backend_checks={len(report.backend_equivalence)} "
+                f"source_checks={len(report.source_equivalence)}"
+            ),
+        )
+    )
+    report.events.append(
+        SmokeEvent(
+            timestamp=datetime.now(timezone.utc).isoformat(),
+            event_type="final_verdict",
+            message=f"verdict pending quality gates",
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)